doc/installation.rst

```diff
@@ -21,6 +21,20 @@ If you are on Debian or Ubuntu Linux we recommend you try
 
 .. _install-packages:
 
+
+Using Anaconda:
+===============
+
+On all platforms, you can use Anaconda_ to install Dipy. To do so issue the following two commands in a terminal::
+
+    conda config --add channels conda-forge
+    conda install dipy
+
+#. Some of the visualization methods require the VTK_ library and this can be installed separately (but only on Python 2)::
+
+    conda install vtk
+
```

Comment:
Ah okay I saw your message. There is a PR that adds it soon. 
Do you know how easy it is to move from conda-forge to the actual conda? For example for vtk you don't need to do ```conda config --add channels conda-forge```.  You can just do ```conda install vtk```. Is it like with neurodebian? For example you start adding packages in neurodebian and then they are moved when ready to debian?